src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/gateways/AssessRisksAndNeedsGateway.kt

```diff
@@ -10,7 +10,9 @@ import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.Response
 import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.UpstreamApi
 import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.UpstreamApiError
 import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.RiskPredictorScore as IntegrationAPIRiskPredictorScore
+import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.Risks as IntegrationApiRisk
```

Comment:
I believe (Not certain) we've used IntegrationAPI opposed to IntegrationApi in the codebase as our alias convention - we should agree upon one and stick to it. I prefer IntegrationApi how you have it.

Reply:
We've used a mix in the codebase. You'll also be glad to hear, we've actually [got a ticket in the backlog](https://dsdmoj.atlassian.net/browse/HIA-426) now to create a convention for these with a few different options (all of which, I believe, are better than either IntegrationAPI/Api).

Reply:
Sensational, thanks for the heads up!